specification/containerinstance/resource-manager/readme.md

```diff
@@ -198,6 +207,10 @@ directive:
   - suppress: UniqueResourcePaths
     from: containerInstance.json
     reason: false positive, see https://github.com/Azure/azure-openapi-validator/issues/176
+suppressions:
+  - code: AvoidAdditionalProperties
+    from: containerInstance.json
+    reason: Using additionalProperties type as the object is user-defined and not subject to any validations at RP level.
```

Comment:
>reason [](http://example.com/codeflow?start=4&length=6)

could you please provide the path also using "where", so that the suppression is only valid in those case and not for whole file.